presto-native-execution/pom.xml

```diff
@@ -359,65 +359,84 @@
-                                        <PRESTO_VERSION>${project.version}</PRESTO_VERSION>
-                                    </buildArgs>
-                                    <writeTestMetadata>false</writeTestMetadata>
+                                    <executable>sh</executable>
+                                    <workingDirectory>${project.parent.basedir}/docker</workingDirectory>
+                                    <arguments>
+                                        <argument>-c</argument>
+                                        <argument>
+                                            <![CDATA[
+                                                 cp ../presto-server/target/presto-server-${project.version}.tar.gz . &&
+                                                 cp ../presto-cli/target/presto-cli-${project.version}-executable.jar . &&
+                                                 docker build -t "presto-coordinator:latest" --build-arg "PRESTO_VERSION=${project.version}" .
+                                                 ]]>
+                                        </argument>
+                                    </arguments>
                                 </configuration>
                             </execution>
+
+                            <!-- Build Presto Native Dependency Docker Image -->
                             <execution>
-                                <id>dependency</id>
+                                <id>build-presto-native-dependency</id>
                                 <phase>install</phase>
                                 <goals>
-                                    <goal>build</goal>
+                                    <goal>exec</goal>
                                 </goals>
                                 <configuration>
-                                    <repository>presto-native-dependency</repository>
-                                    <tag>latest</tag>
-                                    <contextDirectory>${project.basedir}</contextDirectory>
-                                    <dockerfile>${project.basedir}/scripts/dockerfiles/ubuntu-22.04-dependency.dockerfile</dockerfile>
-                                    <buildArgs>
-                                        <BUILD_TYPE>Release</BUILD_TYPE>
-                                    </buildArgs>
-                                    <writeTestMetadata>false</writeTestMetadata>
+                                    <executable>sh</executable>
+                                    <workingDirectory>${project.basedir}</workingDirectory>
+                                    <arguments>
+                                        <argument>-c</argument>
+                                        <argument>
+                                            <![CDATA[
+                                                 docker build -t "presto-native-dependency:latest" \
+                                                 --build-arg BUILD_TYPE="Release" \
```

Comment:
Yes. It is possible to pass properties using fabric8 `docker maven plugin`. Previously we set aside fabric8 plugin because it had issues while passing build args. I found the issue is fixed in latest [fabric8 plugin](https://github.com/fabric8io/docker-maven-plugin). Reworked.